### PR TITLE
fix(deadline-date): date formatting added

### PR DESCRIPTION
**Pull request by mmuchlinski**
## Changes

- Installed 'nunjucks-date-filter' dependency.
- Used the 'date' filter to format the date on the 'expired' page.

### CODE REVIEW

**Review by jmorgan74 — DISMISSED** [review]
👍

File: package.json [hunk 1]
```diff
@@ -21,7 +21,6 @@
     "url": "https://github.com/foundry4/appeal-planning-decision/issues"
   },
   "homepage": "https://github.com/foundry4/appeal-planning-decision#readme",
-  "dependencies": {},
```
Comment (MrSimonEmms) [inline package.json:24]: Please restore this line

File: package-lock.json [hunk 2]
```diff
@@ -857,6 +857,11 @@
         "through": ">=2.2.7 <3"
       }
     },
+    "a-sync-waterfall": {
```
Comment (MrSimonEmms) [inline package-lock.json:860]: I wouldn't have expected any changes in this file - can you restore the original file and then run `make uninstall` and then `make install`. If it does make any changes, that's fine but please confirm this

**Review by MrSimonEmms — CHANGES_REQUESTED** [review]
Root level changes that I'm not expecting have happened

**Comment by MrSimonEmms** [comment]
Can we also have a linear-commit history? It would be better to rebase this commit to the second "remove" didn't exist (it also breaks things because it's required in semantic-release)

Also, please don't make manual changes in package-lock.json - this is an autogenerated file.

image

File: package.json (hunk 1 above)
Comment (jmorgan74) [reply package.json]: done

**Review by jmorgan74 — APPROVED** [review]

**Review by MrSimonEmms — APPROVED** [review]

**Review by MrSimonEmms — APPROVED** [review]

**Pull request merged**